data/reusables/actions/jobs/section-defining-outputs-for-jobs.md

```diff
@@ -40,3 +34,49 @@ jobs:
 ```
 
 {% endraw %}
+
+{% note %}
```

Comment:
This is a good example and deserves its own heading instead of just a note 👍 Perhaps "Using Job Outputs in a Matrix Job"

Reply:
Not sure if I understand it correctly. You suggest to create new separate heading to illustrate how using job outputs works with matrix and put almost all my change in there?

Reply:
Yep, just create a new heading and then everything you added can go under it

Reply:
Done